shardingsphere-operator/api/v1alpha1/shardingsphere_chaos_types.go

```diff
@@ -34,14 +36,201 @@ type ShardingSphereChaosList struct {
 type ShardingSphereChaos struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              ShardingSphereChaosSpec   `json:"spec,omitempty"`
+	Status            ShardingSphereChaosStatus `json:"status,omitempty"`
+}
 
-	Spec ShardingSphereChaosSpec `json:"spec,omitempty"`
+// ShardingSphereChaosSpec defines the desired state of ShardingSphereChaos
+type ShardingSphereChaosSpec struct {
+	InjectJob  JobSpec   `json:"injectJob,omitempty"`
+	ChaosKind  ChaosKind `json:"chaosKind,omitempty"`
```

Comment:
Consider remove this `ChaosKind` and we have already make it explicit in `EmbedChaos`

Reply:
solved